atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -411,7 +412,8 @@ private void saveFinalSweepResults(TableToSweep tableToSweep, SweepResults sweep
             return null;
         });
 
-        sweepMetrics.recordMetrics(tableToSweep.getTableRef(), sweepResults);
+        sweepMetrics.examinedCells(tableToSweep.getTableRef(), sweepResults.getCellTsPairsExamined());
```

Comment:
Yeah I moved the responsibility to the sweeper because the new metrics I added aren't available in the sweep results. Happy to change if you think there's a cleaner way.